### PR TITLE
PT-2391: Clicking N/A on a dropdown prenatal phenotype doesn't do the same on the duplicate

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by veronikaslc** [comment]
@sdumitriu This is better fix for PT-2391, I figured, value identifies picker uniquely enough so additional checks that were causing complications and bug are unnecessary.

File: components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml [hunk 1]
```diff
@@ -1778,11 +1778,7 @@
     var newItem = this.list ? this.list.down('input[id="' + id.replace(/[^a-zA-Z0-9_-]/g, '\\$&amp;') + '"]') : $(id);
     //this.ensureVisible(newItem);
     this.synchronizeSelection(newItem);
-    newItem.observe('change', this.synchronizeSelection.bind(this, newItem));
     this.updateListTools();
-    Event.fire(document, 'xwiki:dom:updated', {'elements' : [listItem]});
-    this.onItemAdded(newItem)
```
Comment (sdumitriu) [inline components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml]: Removing this causes the category to not be correct.

Comment (veronikaslc) [inline components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml]: right, will remove this commit from PR

**Comment by veronikaslc** [comment]
@sdumitriu removed unnecessary commit from PR

**Comment by sashaandjic** [comment]
tested. defect corrected. problems that @sdumitriu reported not found.

**Comment by veronikaslc** [comment]
reabased

**Comment by sashaandjic** [comment]
retested

**Comment by veronikaslc** [comment]
reabased, @sashaandjic may be needs retesting again..

**Comment by sashaandjic** [comment]
retested. all good.

**Pull request merged**